x-pack/plugin/security/qa/multi-cluster/src/javaRestTest/java/org/elasticsearch/xpack/remotecluster/RemoteClusterSecurityRestIT.java

```diff
@@ -84,6 +85,17 @@ public class RemoteClusterSecurityRestIT extends AbstractRemoteClusterSecurityTe
             .build();
     }
 
+    /**
+     * This is needed because `randomBoolean` is not available inside the outer `static {}` scope,
```

Comment:
A quirk here: using `randomBoolean()` in the `static {}` block results in a `classMethod` exception:

```
No context information for thread: Thread[id=32, name=SUITE-RemoteClusterSecurityRestIT-seed#[85BC9D463CA78D4], state=RUNNABLE, group=TGRP-RemoteClusterSecurityRestIT]. Is this thread running under a class com.carrotsearch.randomizedtesting.RandomizedRunner runner context? Add @RunWith(class com.carrotsearch.randomizedtesting.RandomizedRunner.class) to your test class. Make sure your code accesses random contexts within @BeforeClass and @AfterClass boundary (for example, static test class initializers are not permitted to access random contexts).
java.lang.IllegalStateException: No context information for thread: Thread[id=32, name=SUITE-RemoteClusterSecurityRestIT-seed#[85BC9D463CA78D4], state=RUNNABLE, group=TGRP-RemoteClusterSecurityRestIT]. Is this thread running under a class com.carrotsearch.randomizedtesting.RandomizedRunner runner context? Add @RunWith(class com.carrotsearch.randomizedtesting.RandomizedRunner.class) to your test class. Make sure your code accesses random contexts within @BeforeClass and @AfterClass boundary (for example, static test class initializers are not permitted to access random contexts).
	at com.carrotsearch.randomizedtesting.RandomizedContext.context(RandomizedContext.java:249)
	at com.carrotsearch.randomizedtesting.RandomizedContext.current(RandomizedContext.java:134)
	at org.apache.lucene.tests.util.LuceneTestCase.random(LuceneTestCase.java:792)
	at org.elasticsearch.test.ESTestCase.randomBoolean(ESTestCase.java:775)
	at org.elasticsearch.xpack.remotecluster.RemoteClusterSecurityRestIT.<clinit>(RemoteClusterSecurityRestIT.java:49)
	at java.base/java.lang.Class.forName0(Native Method)
	at java.base/java.lang.Class.forName(Class.java:496)
	at java.base/java.lang.Class.forName(Class.java:475)
	at com.carrotsearch.randomizedtesting.RandomizedRunner$2.run(RandomizedRunner.java:631)
```

"for example, static test class initializers are not permitted to access random contexts" explains this.

We could probably come up with a cleaner workaround but that requires some work. I think my hack is sufficient for now.